lib/libvarnishapi/vsl.c

```diff
@@ -346,6 +346,8 @@ VSL_PrintTransactions(struct VSL_data *vsl, struct VSL_transaction * const pt[],
 			case VSL_t_req:
 				if (!vsl->c_opt)
 					continue;
+				if (t->reason == VSL_r_esi && !vsl->E_opt)
```

Comment:
Won't this change `varnishlog -c` to not any longer display ESI by default?

Reply:
Yes, it is mentioned explicitly as a breaking change in case Someone:tm: would like to bring some patches to a stable branch: 65503181e6b479538698bcc7ef8e7323e13f80ec.

My reasoning is that ESI requests are not client requests, but requests coming from Varnish itself, so it's ok to leave them out of `-c` and grant them a new `-E`.

Reply:
I agree with this